__tests__/app.test.js

```diff
@@ -29,3 +29,61 @@ describe("GET /api/topics", () => {
         })
     })
 })
+
+describe("GET /api/articles/:article_id", () => {
+    test('returns article by id with correct properties', () => {
+        return request(app).get('/api/articles/2')
+        .expect(200)
+        .then(({body}) => {
+            const articles = body.article
+            articles.forEach((article) => {
+                expect(article).toMatchObject({
+                    article_id: expect.any(Number),
```

Comment:
Here we specifically want the article_id to be 2, this will prove to us we are getting the correct article back